### ISSUE TITLE
Warning is soap.ol test

### ISSUE THREAD

**Issue by klag** [jolie/jolie#302]
if you run the test soap.ol, the following warnings appear

INFORMAZIONI: Metro monitoring rootname successfully set to: com.sun.metro:pp=/,type=WSEndpoint,name=CalcService-CalcServicePort
mar 20, 2021 10:28:56 PM com.sun.xml.ws.server.WSEndpointImpl closeManagedObjectManager
INFORMAZIONI: Closing Metro monitoring root: com.sun.metro:pp=/,type=WSEndpoint,name=CalcService-CalcServicePort
passed.

**Comment by thesave** [comment]
That's new to me.
It seems related to https://github.com/jolie/jolie/pull/299 — although that seems INFO level and not WARNING.
You get that running `jolie test extensions/soap.ol` right? 
Under what OS and JRE version?

**Comment by klag** [comment]
Yes. I am using an Ubuntu 20 (but it happens also with Ubuntu 18). 
It happens with both Java 8 and 11

**Comment by thesave** [comment]
Ok, a quick-and-dirty solution could be to try and suppress the WARNINGs (you got those at https://github.com/jolie/jolie/pull/299 under j11 but not under j8, right?) and INFO messages of the metro logger in the `soap.ol` test (the JavaService, more precisely).

The clean solution would be to replace the 

```xml
        <dependency>
            <groupId>com.sun.xml.ws</groupId>
            <artifactId>rt</artifactId>
            <version>2.3.3</version>
        </dependency>
```
we have [here](https://github.com/jolie/jolie/blob/fa61d55ae95599194361062eeb5134a1feaae039/test/extensions/private/WS-test/pom.xml#L119) but I could not find any alternative that works.
The other clean solution is to rewrite WSTest to use some other library for SOAP. I don't have the time for that, but I can give pointers if someone wants to tackle it :)

**Comment by klag** [comment]
Since it is just for a test, I think it is sufficient to suppress the warnings

**Comment by thesave** [comment]
I tried looking for some way to suppress the metro monitor INFOs but [AFAIU](https://javaee.github.io/metro/doc/user-guide/ch20.html) it seems doable only passing the parameter `-Dcom.sun.xml.ws.monitoring.endpoint=false` to the JVM, but I don't know how we can manage to do it.
The other WARNINGs come from Java 9+. We can suppress those via some unsafe reflection, but I prefer not to mess with that :D
Ideas are welcome :)

**Comment by mwallnoefer** [comment]
> I tried looking for some way to suppress the metro monitor INFOs but [AFAIU](https://javaee.github.io/metro/doc/user-guide/ch20.html) it seems doable only passing the parameter `-Dcom.sun.xml.ws.monitoring.endpoint=false` to the JVM, but I don't know how we can manage to do it. The other WARNINGs come from Java 9+. We can suppress those via some unsafe reflection, but I prefer not to mess with that :D Ideas are welcome :)

Maybe thinking the other way around: do we need these messages at all? Otherwise we may simply change the launchers.

**Comment by mwallnoefer** [comment]
@thesave 's workaround suggesting to add additional JVM options in the launcher seems perfectly valid. So I guess that it makes little sense to keep the bug open.
